### ISSUE TITLE
The table `public.<TableName>` does not exist in the current database.

### ISSUE THREAD

**Issue by florianmartens** [prisma/prisma#10633]
### Bug description

I'm encountering some weird behavior with Prisma seed. At this point, I can only assume that it is a bug within the client. When resetting the database with

`yarn workspace @nl/server dotenv -e .env.development prisma migrate reset`

I receive the output:

>  9 
>   10 async function seed() {
>   11     // ✅ generate root organization
> → 12     const rootOrganization = await prisma.organization.create(
>   The table `public.Organization` does not exist in the current database.
>     at cb (/Users/.../Code/nest-boilerplate/packages/server/node_modules/@prisma/client/runtime/index.js:38683:17)
>     at seed (/Users/.../Code/nest-boilerplate/packages/server/prisma/seed.ts:12:30) {
>   code: 'P2021',
>   clientVersion: '3.6.0',
>   meta: { table: 'public.Organization' }
> }
> 
> An error occurred while running the seed command:
> Error: Command failed with exit code 1: ts-node prisma/seed.ts

As is the creation of the DB schema is left completely to Prisma. So I'm surprised to see that Prisma tries to run the `seed` script before the schema seems to be created. If I run `seed` after the reset is complete as a standalone command, everything works.

Sidenote: I'm working in development and have deleted all migrations (maybe this plays a part?).

### How to reproduce

Run Prisma `prisma migrate reset`

### Prisma information

```
model Organization {
  id              String    @id @default(cuid())
  isPrivatePerson Boolean   @default(false)
  name            String
  slug            String    @unique
  createdAt       DateTime? @default(now())
  updatedAt       DateTime? @default(now())
  userList        User[]
  addressLineOne  String?
  addressLineTwo  String?
  zipCode         String?
  city            String?
  state           String?
  country         String?

  stripeCustomerId         String?
  stripeSubscriptionId     String?
  stripeCurrentProductName String?
  currentBillingPeriodEnds DateTime?
  paymentStatus            PaymentStatusTypes?
  licenseOverwrite         LicenseOverwrite?
  isSetup                  Boolean             @default(false)

  workspaceList              Workspace[]
  organizationInvitationList OrganizationInvitation[]
  contactList                Contact[]
  squadList                  Squad[]
  postList                   Post[]
  digestList                 Digest[]
}
```

### Environment & setup

- MacOS
- PostgreSQL
- Node 14.16.1

### Prisma Version

"@prisma/client": "^3.5.0",
"prisma": "^3.5.0",

**Comment by janpio** [comment]
> Sidenote: I'm working in development and have deleted all migrations (maybe this plays a part?).

`migrate reset` resets the database by putting it into the state represented by the Migration files. If there are none, the database will be empty. If you just want to get the database onto the state of your current schema file, you should probably use `db push`.

**Comment by radeon8** [comment]
I have the same issue, but it says ``The table main.<TableName> does not exist in the current database.``

**Comment by radeon8** [comment]
Please try to change the database to Mysql

**Comment by davidjgonzalez** [comment]
I am getting the same w/ SQLLite ... If i look at the SQLite DB, the tables area all there - i even manually added a row into the table in question, and i can run the exact query that is complaining in my SQLLite tool. 
This works fine if i point it at my PostgreSQL ... but id like to use SQLLite for dev. What could be causing this?

**Comment by AndresdoSantos** [comment]
I ran an `npx prisma migrate reset` and it worked.

**Comment by pavelsvitek** [comment]
I resolved the problem after changing database to PostgreSQL. Which indicates that there's some problem in the SQLite "dialect" which results in prefix "main." in SQL query, but technically the prefix either should not be there or should be different.

**Comment by Weakky** [comment]
Hey @florianmartens,

Could you please provide us with either a small reproduction repo or, a complete seed script that leads to the error you're encountering? That'd help us a lot.

Thank you 🙏

**Comment by rodrigo-calle** [comment]
**Environment** 

Ubuntu
PostgreSQL
Node 16.20.0

Using the following command to map my model in my database solved the error for me:

`npx prisma migrate dev --name init`

**Comment by jharrell** [comment]
@florianmartens 

Hi! Following up here. Did you see @Weakky or @rodrigo-calle 's comments? Are you still experiencing this issue?

Thank you!

**Comment by jharrell** [comment]
Hi there, 

Without further info we'll have to close this issue. Please feel free to drop a comment or open a new issue with the info requested above.

Thank you!

**Comment by iovoz** [comment]
I also met this error.
Invalid `prisma.shopifyBigCCustomer.create()` invocation:
The table `public.ShopifyBigCCustomer` does not exist in the current database.
But the table is actually there, see my screenshot below.
![image](https://github.com/prisma/prisma/assets/18189938/6cff8cc2-ee71-48a4-b07a-4e3cd8a06511)

**Comment by pos1234** [comment]
`npx prisma migrate dev` this worked for me

**Comment by saketkum11** [comment]
i am getting same problem

**Comment by igortas** [comment]
I'm getting the same problem,
It's not working even after migrate dev and generate commands.

Model is using @@map, and running from Nest.JS throws error that table can't be found:

```The table `public.xxx` does not exist in the current database```

**Comment by welshm** [comment]
I am getting this problem when using SQLite

**Comment by yokohailemariam** [comment]
if you are working with nestjs make sure you have `import 'dotenv/config'; `at your main.ts file

**Comment by Nick-Lemy** [comment]
> import 'dotenv/config'

I even tried to put the DATABASE_URL strings directly in the prisma service but still not working 💔

**Comment by Nick-Lemy** [comment]
After more than 24hours of debugging, I was using NestJS and in the prisma service file, when instantiating the adapter, in `PrismaPg` I added the db url using a property called `url` which I got from Nest docs while it should be `connectionString`. 



**TL;DR: switching `url` to `connectionString` worked.**

## Before

<img width="607" height="258" alt="Image" src="https://github.com/user-attachments/assets/dec9216d-934f-46af-940b-268891ad0a67" />

## After

<img width="595" height="298" alt="Image" src="https://github.com/user-attachments/assets/87210768-41b5-4b54-9ca5-dda542b2cd5e" />
